app/models/dependent.rb

```diff
@@ -92,24 +92,24 @@ class Dependent < ApplicationRecord
   validates :ip_pin, ip_pin: true
 
   QUALIFYING_CHILD_RELATIONSHIPS = [
```

Comment:
The casing on these relationships will cause mismatches. The relationships get stored in the database as all lowercased strings (see relationship helper spec for what the values are). We send them all to the IRS as upper case strings.

Choosing to upcase vs downcase was arbitrary.